group_vars/sn07.yml

```diff
@@ -350,7 +350,7 @@ galaxy_venv_dir: "{{ galaxy_root }}/venv"
 galaxy_job_working_directory: "{{ galaxy_config['galaxy']['job_working_directory'] }}"
 ucsc_build_sites:
 
-galaxy_virtualenv_command: "{{ conda_prefix }}/envs/_galaxy_/bin/virtualenv"
+galaxy_virtualenv_command: "python -m venv"
```

Comment:
Given that venv links to system Python

```shell
[jose@BILP51 Downloads]$ python -m venv test_venv
[jose@BILP51 Downloads]$ ls -lah test_venv/bin
insgesamt 48K
drwxr-xr-x 1 jose jose  164 29. Aug 10:59 .
drwxr-xr-x 1 jose jose   56 29. Aug 10:59 ..
-rw-r--r-- 1 jose jose 2,0K 29. Aug 10:59 activate
-rw-r--r-- 1 jose jose  933 29. Aug 10:59 activate.csh
-rw-r--r-- 1 jose jose 2,2K 29. Aug 10:59 activate.fish
-rw-r--r-- 1 jose jose 8,9K 29. Aug 10:59 Activate.ps1
-rwxr-xr-x 1 jose jose  246 29. Aug 10:59 pip
-rwxr-xr-x 1 jose jose  246 29. Aug 10:59 pip3
-rwxr-xr-x 1 jose jose  246 29. Aug 10:59 pip3.11
lrwxrwxrwx 1 jose jose   15 29. Aug 10:59 python -> /usr/bin/python
lrwxrwxrwx 1 jose jose    6 29. Aug 10:59 python3 -> python
lrwxrwxrwx 1 jose jose    6 29. Aug 10:59 python3.11 -> python
```

I'd feel safer using 

`galaxy_virtualenv_command: "{{ conda_prefix }}/envs/_galaxy_/bin/python -m venv"`

Reply:
Who is updating `{{ conda_prefix }}/envs/_galaxy_/` is that 3.11 then?

Reply:
@sanjaysrikakulam You're more familiar with the miniconda role.

Reply:
Definitely yes.

```text
[jose@BILP51 Downloads]$ conda create -n python_36 python=3.6
[...]
[jose@BILP51 Downloads]$ /home/jose/.mambaforge/envs/python_36/bin/python -m venv venv_python_36
[jose@BILP51 Downloads]$ venv_python_36/bin/python --version
Python 3.6.15

```